Update Firebase distribution information regarding artifact preference (#676)
codemagic-ci-cd/codemagic-docs: content/yaml-publishing/distribution.md

#### Publishing only the Android app bundle or APK artifact to Firebase App Distribution

> If you are building both an Android app bundle and an APK in your workflow, Codemagic will, by default, try to publish the bundle to Firebase App Distribution. If you wish to publish the APK, specify the artifact type as `apk` using the `artifact_type` field. For more clarity, this can also be set to `aab` for the bundle:

Reviewer: ```suggestion
If you are building both an Android app bundle and an APK in your workflow, Codemagic will, by default, try to publish the bundle to Firebase App Distribution. If you wish to publish the APK, specify the artifact type as `apk` using the `artifact_type` field. For clarity, `artifact_type` can also be also set to `aab` for publishing the app bundle.
```
